src/app/pages/component-viewer/component-viewer.scss

```diff
@@ -9,13 +15,91 @@
 }
 
 .docs-component-viewer-header {
-  margin: 50px;
+  margin-left: 20px;
+
+  h1 {
+    font-size: 30px;
+    font-weight: 300;
+    margin: 0;
+    padding: 50px;
+  }
 }
 
 .docs-component-viewer-tabbed-content {
-  margin: 50px;
+  margin: 50px 70px;
 
   md-tab {
     margin: 30px;
   }
+
+  h1, h4 {
+    display:inline-block;
+    font-size: 20px;
+    font-weight: 400;
+    padding: 5px;
+  }
+
+  h3, h2 {
+    font-size: 20px;
+    font-weight: 400;
+    margin-bottom: 30px;
+    margin-top: 80px;
+  }
+
+  h5 {
+    font-size: 16px;
+  }
+
+  .md-tab-body-wrapper h2{
+    margin-top: 0px;
+  }
+
+  p {
+    font-size: 16px;
+    line-height: 28px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  table {
+    border-collapse: collapse;
+    border-radius: 2px;
+    border-spacing: 0;
+    margin: 0 0 32px 0;
+    width: 100%;
+  }
+
+  table tbody th{
+    max-width: 100px;
+    padding: 13px 32px;
+    text-align: left;
+  }
+
+  td {
+    font-weight: 400;
+    padding: 0 32px !important;
+  }
+
+  th {
+    font-size: 16px;
+    font-weight: 400;
+  }
+}
+
+.md-tab-header {
+  border: none;
+  margin-bottom: 30px;
+}
+
+.md-tab-label {
```

Comment:
Make this a sub-ruleset of .md-tab-header, and make .md-tab-header a sub-ruleset of .docs-component-viewer-tabbed-content. This will increase the specificity enough that you can remove the !importants.